src/engraving/libmscore/tremolo.cpp

```diff
@@ -420,9 +420,6 @@ void Tremolo::layoutTwoNotesTremolo(double x, double y, double h, double spatium
     double x2 = _chord2->stemPosBeam().x();
     if (!stem2 && _chord2->up()) {
         double nhw = score()->noteHeadWidth();
-        if (_chord2->noteType() != NoteType::NORMAL) {
-            nhw *= score()->styleD(Sid::graceNoteMag);
-        }
         nhw *= _chord2->mag();
         x2 -= nhw;
```

Comment:
I'd maybe simplify this to either
```c++
x2 -= score()->noteHeadWidth() * _chord2->mag();
```
or
```c++
x2 -= _chord2->noteHeadWidth();
```

Reply:
Good idea! I made `Chord::noteHeadWidth` public and just used that.